components/zcash_address/src/kind/unified.rs

```diff
@@ -3,27 +3,11 @@ use std::collections::HashSet;
 use std::convert::{TryFrom, TryInto};
 use std::error::Error;
 use std::fmt;
-use std::io::Write;
 use zcash_encoding::CompactSize;
 
-use crate::kind;
+pub(crate) mod address;
 
-/// The HRP for a Bech32m-encoded mainnet Unified Address.
-///
-/// Defined in [ZIP 316][zip-0316].
-///
-/// [zip-0316]: https://zips.z.cash/zip-0316
-pub(crate) const MAINNET: &str = "u";
-
-/// The HRP for a Bech32m-encoded testnet Unified Address.
-///
-/// Defined in [ZIP 316][zip-0316].
-///
-/// [zip-0316]: https://zips.z.cash/zip-0316
-pub(crate) const TESTNET: &str = "utest";
-
-/// The HRP for a Bech32m-encoded regtest Unified Address.
-pub(crate) const REGTEST: &str = "uregtest";
+pub(crate) use address::Address;
```

Comment:
Why is this `pub(crate)` instead of `pub`? The `Address` type is needed in the zcashd ffi, and I don't see how to get to it.

Reply:
Yeah, these types were previously visible in this module. I think both `Address` and `Receiver` should be `pub use`d here to retain their previous visibility.